test/ffmpeg-vaapi/encode/encoder.py

```diff
@@ -107,19 +117,31 @@ def encode(self):
 
     vars(self).update(rcmodeu = self.rcmode.upper())
 
-    self.encoded = get_media()._test_artifact(
-      "{}.{}".format(self.gen_name(), self.get_file_ext()))
-
-    iopts = self.gen_input_opts()
-    oopts = self.gen_output_opts()
-
-    self.output = call(
-      "ffmpeg -hwaccel vaapi -vaapi_device /dev/dri/renderD128 -v verbose"
-      " {iopts} {oopts}".format(iopts = iopts, oopts = oopts))
-
-    self.check_output()
-    self.check_bitrate()
-    self.check_metrics()
+    if vars(self).get("r2r", None) is not None:
+      assert (self.r2r > 1) is True, "invalid r2r value"
```

Comment:
`assert type(self.r2r) is int and self.r2r > 1, "invalid r2r value"`

Reply:
modified as for comment